living_documentation_generator/action_inputs.py

```diff
@@ -125,10 +125,15 @@ def validate_inputs(out_path: str) -> None:
         # Check that the INPUT_OUTPUT_PATH is not a project directory
         # Note: That would cause a rewriting project files
         project_directories = get_all_project_directories()
-        if DEFAULT_OUTPUT_PATH in project_directories:
-            project_directories.remove(DEFAULT_OUTPUT_PATH)
+        default_abspath_output_path = os.path.abspath(DEFAULT_OUTPUT_PATH)
```

Comment:
Why do you here work with constant value instead of user's input?

Reply:
I worked with constant default, because the file can be created without user knowing. I still think, it should be kept there. So first option is to check as well if there is already an user's input OR you mentioned few weeks ago, that there is no point to remove these files at all, since it will be run only once on the GH everytime.

Reply:
After the discussion, the change is fixed in commit: [af039ef](https://github.com/AbsaOSS/living-doc-generator/pull/43/commits/af039ef6c5c084aa11986ddc256ec4311d3643a4).